doc/reference/power_management/system-pm.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There are some style issues in tge diagram:

![image](https://user-images.githubusercontent.com/25011557/119948910-9d9ac700-bf99-11eb-869a-5f573eb071d7.png)